dfastmi/gui/dialog_view.py

```diff
@@ -192,16 +192,22 @@ def _update_reach(self, data):
         self._update_condition_files()
 
     def _update_condition_files(self):
-        """Update the condition files."""
+        """
```

Comment:
@ralph-peelen-deltares I would like to discuss these changes with you when you are available.

Reply:
See PR #100 and #101, no need to have a call to load configuration, this is part of viewmodel responsibility.
When instantiating the view, the data should be retrieved from the already instantiated viewmodel. With corrected eventing, blocking of emits is not needed.

Reply:
Discussed with Bert.